mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -665,152 +683,184 @@ public Future<Integer> calculateGeneration(ReactiveClassicGenericQueryExecutor t
 
   @Override
   public Future<ParsedRecord> updateParsedRecord(Record record, String tenantId) {
```

Comment:
Hello @okolawole-ebsco, looks like we don't need to wrap the updateParsedRecord() method code with executeBlocking because the `transaction()` method is asynchronous and does not perform blocking operations

Reply:
Good point. I have reverted that method.